worksheets_test.go

```diff
@@ -84,6 +84,24 @@ func (s *Zuite) TestNewDefinitionsErrors() {
 		`worksheet refs_to_worksheet {
 			89:refs_here [][]some_other_worksheet
 		}`: `refs_to_worksheet.refs_here: unknown worksheet some_other_worksheet referenced`,
+
+		`worksheet constrained_and_computed {
+			1:age number[0]
+			69:some_field text constrained_by { return true } computed_by { return age + 2 }
+		}`: `expected index, found computed_by`,
```

Comment:
To my earlier point about using `peekWithChoice`: the fact that you have two different error messages depending on the order in which you specify `constrained_by` and  `computed_by` on the same field leaks the implementation a bit. It shows an asymmetry in the handling in a way.